cypress/support/index.js

```diff
@@ -18,3 +18,9 @@ import './commands'
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+    // returning false here prevents Cypress from
+    // failing the test
+    return false
+  })
```

Comment:
This is to avoid app errors failing the tests, e.g.:
![Screenshot from 2022-01-12 15-19-16](https://user-images.githubusercontent.com/77306764/149568247-52d96a5e-4f7e-4543-8575-a9000ef6db26.png)
I've added a tech debt issue: